packages/deno/src/integrations/deno-cron.ts

```diff
@@ -5,7 +5,74 @@ import type { DenoClient } from '../client';
 type CronOptions = { backoffSchedule?: number[]; signal?: AbortSignal };
 type CronFn = () => void | Promise<void>;
 // Parameters<typeof Deno.cron> doesn't work well with the overloads 🤔
-type CronParams = [string, string, CronFn | CronOptions, CronFn | CronOptions | undefined];
+type CronParams = [string, string | Deno.CronSchedule, CronFn | CronOptions, CronFn | CronOptions | undefined];
+
+/**
+ * Copied from Deno source:
+ * // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+ * https://github.com/denoland/deno/blob/cd480b481ee1b4209910aa7a8f81ffa996e7b0f9/ext/cron/01_cron.ts
+ */
+function formatToCronSchedule(
```

Comment:
Let's move the vendored functions to another file, and copy the entire MIT license.